bindings/cs/MongoDB.Libmongocrypt/CryptOptions.cs

```diff
@@ -30,27 +30,33 @@ public class CryptOptions
         public byte[] EncryptedFieldsMap { get; }
         public IReadOnlyList<KmsCredentials> KmsCredentials { get; }
         public byte[] Schema { get; }
+        public string CsfleLibPath { get; }
```

Comment:
Let's place in in alphabetical order too

Reply:
Done.